features/support/usageSteps.js

```diff
@@ -72,33 +70,34 @@ Then(/^the scenario buttons for the (first|second) feature (?:are|will be) expan
   const index = featureIndex === 'first' ? 0 : 1;
   const featureButtons = this.outputHTML.getElementsByClassName('feature-button');
   const selectedFeatureButton = featureButtons[index];
-  // eslint-disable-next-line no-unused-expressions
   expect(selectedFeatureButton.classList.contains('active')).to.be.true;
   const scenarioPanel = selectedFeatureButton.nextElementSibling;
-  // eslint-disable-next-line no-unused-expressions
   expect(scenarioPanel.style.maxHeight).to.not.be.empty;

```

Comment:
should we switch from expect to should? that might make it a bit clearer

Reply:
Okay, so I have been reading about the Should notation out here: https://www.chaijs.com/guide/styles/. I am not familiar with Object.prototype, so maybe I am missing something, but how am I supposed to setup the should() function on the various objects I need to assert when I do not have references to those objects ahead of time (when I am requiring the should like they do in the example....)?

Reply:
```js
const chai = require('chai');
chai.should();
```

That's all you need to do to set up should.

Reply:
Hmm, okay so I tried this and the tests ran.  But the `no-unused-expressions` linting errors remained.  Basically I did this:
```.js
const should = require('chai').should();
selectedFeatureWrapper.classList.contains('active').should.be.true;
```

The first line has the `no-unused-vars` warning and the second line has the `no-unused-expressions` warning.

Reply:
ah, i forgot about this in other projects. i typically apply a different set of lintings for chai...
[like this](https://github.com/WFCD/warframe-worldstate-data/blob/master/test/.eslintrc.json)

I forgot that we basically overrided a bit so it worked right

Reply:
so we can just keep using expect and not worry about the linting bits, basically.

Reply:
Done